packages/jekyll-format/jekyll-format.0.3.0/opam

```diff
@@ -0,0 +1,53 @@
+opam-version: "2.0"
+synopsis: "Jekyll post parsing library"
+description: """
+This library provides an OCaml interface to parsing
+posts in the Jekyll format."""
+maintainer: ["Anil Madhavapeddy <anil@recoil.org>"]
+authors: ["Anil Madhavapeddy" "Patrick Ferris"]
+license: "MIT"
+homepage: "https://github.com/avsm/jekyll-format"
+bug-reports: "https://github.com/avsm/jekyll-format/issues"
+depends: [
+  "dune" {>= "2.8"}
+  "ocaml" {>= "4.05.0"}
+  "yaml-sexp" {>= "3.0.0"}
+  "yaml" {>= "3.0.0"}
+  "sexplib"
+  "sexplib0"
```

Comment:
```suggestion
```
This is only required for `yaml-sexp`:
```
#=== ERROR while compiling yaml-sexp.3.0.0 ====================================#
# context              2.1.0 | linux/x86_64 | ocaml-base-compiler.4.05.0 | file:///home/opam/opam-repository
# path                 ~/.opam/4.05/.opam-switch/build/yaml-sexp.3.0.0
# command              ~/.opam/opam-init/hooks/sandbox.sh build dune build -p yaml-sexp -j 31
# exit-code            1
# env-file             ~/.opam/log/yaml-sexp-23-06b106.env
# output-file          ~/.opam/log/yaml-sexp-23-06b106.out
### output ###
#       ocamlc lib_sexp/.yaml_sexp.objs/byte/yaml_sexp.{cmo,cmt} (exit 2)
# (cd _build/default && /home/opam/.opam/4.05/bin/ocamlc.opt -w -40 -w -9-27-32 -g -bin-annot -I lib_sexp/.yaml_sexp.objs/byte -I /home/opam/.opam/4.05/lib/base -I /home/opam/.opam/4.05/lib/base/caml -I /home/opam/.opam/4.05/lib/base/shadow_stdlib -I /home/opam/.opam/4.05/lib/bytes -I /home/opam/.opam/4.05/lib/ctypes -I /home/opam/.opam/4.05/lib/integers -I /home/opam/.opam/4.05/lib/result -I /home/opam/.opam/4.05/lib/rresult -I /home/opam/.opam/4.05/lib/sexplib -I /home/opam/.opam/4.05/lib/yaml -I /home/opam/.opam/4.05/lib/yaml/bindings -I /home/opam/.opam/4.05/lib/yaml/bindings/types -I /home/opam/.opam/4.05/lib/yaml/c -I /home/opam/.opam/4.05/lib/yaml/ffi -I /home/opam/.opam/4.05/lib/yaml/types -intf-suffix .ml -no-alias-deps -o lib_sexp/.yaml_sexp.objs/byte/yaml_sexp.cmo -c -impl lib_sexp/yaml_sexp.pp.ml)
# File "lib_sexp/yaml_sexp.ml", line 16, characters 5-23:
# Error: Unbound module Sexplib0
#     ocamlopt lib_sexp/.yaml_sexp.objs/native/yaml_sexp.{cmx,o} (exit 2)
# (cd _build/default && /home/opam/.opam/4.05/bin/ocamlopt.opt -w -40 -w -9-27-32 -g -I lib_sexp/.yaml_sexp.objs/byte -I lib_sexp/.yaml_sexp.objs/native -I /home/opam/.opam/4.05/lib/base -I /home/opam/.opam/4.05/lib/base/caml -I /home/opam/.opam/4.05/lib/base/shadow_stdlib -I /home/opam/.opam/4.05/lib/bytes -I /home/opam/.opam/4.05/lib/ctypes -I /home/opam/.opam/4.05/lib/integers -I /home/opam/.opam/4.05/lib/result -I /home/opam/.opam/4.05/lib/rresult -I /home/opam/.opam/4.05/lib/sexplib -I /home/opam/.opam/4.05/lib/yaml -I /home/opam/.opam/4.05/lib/yaml/bindings -I /home/opam/.opam/4.05/lib/yaml/bindings/types -I /home/opam/.opam/4.05/lib/yaml/c -I /home/opam/.opam/4.05/lib/yaml/ffi -I /home/opam/.opam/4.05/lib/yaml/types -intf-suffix .ml -no-alias-deps -o lib_sexp/.yaml_sexp.objs/native/yaml_sexp.cmx -c -impl lib_sexp/yaml_sexp.pp.ml)
# File "lib_sexp/yaml_sexp.ml", line 16, characters 5-23:
# Error: Unbound module Sexplib0
```

Reply:
woops indeed, that was the entire point of the package split :-)